.circleci/config.yml

```diff
@@ -15,3 +15,30 @@ jobs:
           if [ "${CIRCLE_BRANCH}" == "master" ]; then
             ./architect deploy
           fi
+
+  e2eTestInformerIntegration:
+    machine: true
+    working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/operatorkit
+    environment:
+      MINIKUBE_VERSION: v0.25.0
```

Comment:
I feel like this should be a default in e2e-harness. We can then switch to newer versions everywhere more easily.

Reply:
@fgimenez do we have this somewhere tracked?

Reply:
Just added it here https://github.com/giantswarm/e2e-harness/issues/29